api.js

```diff
@@ -228,17 +233,19 @@ RosterAPI.prototype.parseAccountAndPin = function(content) {
 
 RosterAPI.prototype.toPhoneContext = function(countryOrPhone) {
 
-    if (countryOrPhone == null)
```

Comment:
Non-nitpick - actually this test is meant to catch both null *and* undefined - understand lots of places make === best practice but I find "== null" useful in cases where we actually don't care about the difference between the two.

If you want to be super explicit about it you could do === null and === undefined - I've never had real issues from this (=== false, and javascript's weird issues with falseness are the ones I'm more anal about b/c they've messed me up in the past).

Reply:
@gregstuderoaf that makes sense, if we actually wanted to catch both null and undefined. 

When would countryOrPhone ever be null, though? That would involve someone explicitly calling `toPhoneContext(null)` ... which doesn't seem likely. Also, in line 248 I throw an error if the `phone` is null -- but I can get rid of that if you don't think it's helpful.

Reply:
discussed in-persoin